docs/troubleshoot/ze/known-ze.md

```diff
@@ -10,6 +10,44 @@ These bidirectional (BiDi) languages are not currently supported in Visual Studi
 
 As a result, VS Code extensions like Zowe Explorer, Zowe Explorer CICS Extension, and Zowe Explorer FTP Extension are not able to support BiDi languages in files.
 
+## Client certificate support
+
+Some Zowe Explorer users prefer to use certificates to access the API Mediation Layer. This can be the case in sites that use credentials such as passwords and multifactor authentication, which are known to expire often. On the other hand, certificates can be valid for much longer.
```

Comment:
Rather than "which are known to expire often", it might be better to say "which may only be valid for a short period of time".